plugins/database/src/low_level_odbc.cpp

```diff
@@ -31,13 +31,22 @@
 #include "irods/private/low_level_odbc.hpp"
 
 #include "irods/irods_log.hpp"
+#include "irods/irods_logger.hpp"
 #include "irods/irods_error.hpp"
 #include "irods/irods_stacktrace.hpp"
 #include "irods/irods_server_properties.hpp"
 
 #include <cctype>
 #include <string>
 
+namespace
+{
+    using log_db = irods::experimental::log::database;
+
+    // NOLINTNEXTLINE(cppcoreguidelines-macro-usage)
+#define IRODS_DB_TRACE(msg, ...) log_db::trace("{}@{}: " msg, __func__, __LINE__ __VA_OPT__(,) __VA_ARGS__);
```

Comment:
and just asking... `#define` has to be on left edge (no indentation), even though it's within the namespace block?

Reply:
No, it's not required. Making preprocessor logic follow its own indentation helps to make it more noticeable, which is good since it's handled at a different stage during compilation.

Preprocessor exists in its own world.

Reply:
This is also the formatting enforced by our clang-format configuration (on purpose).

Reply:
Correct.

@trel Does that answer your question?

Reply:
it does.   resolved!